docs/tutorials/dqn.rst

```diff
@@ -123,9 +123,9 @@ Tianshou provides :class:`~tianshou.trainer.onpolicy_trainer` and :class:`~tians
         policy, train_collector, test_collector,
         max_epoch=10, step_per_epoch=1000, collect_per_step=10,
         episode_per_test=100, batch_size=64,
-        train_fn=lambda e: policy.set_eps(0.1),
-        test_fn=lambda e: policy.set_eps(0.05),
-        stop_fn=lambda x: x >= env.spec.reward_threshold,
+        train_fn=lambda epoch, num_env_step: policy.set_eps(0.1),
```

Comment:
IMHO `env_step_idx` would be clearer

Reply:
usually, `num` is for total/cumulative sum. That's why I think `idx` is more appropriate.

Reply:
How about simply use "step", just like "epoch".

Reply:
My original proposal is "env_step" because "step" may be ambiguity. Since we have two kinds of step: env step and gradient step.

Reply:
"env_step" is okay then. Be sure to explain it in the doc.